site/src/main/java/uk/nhs/digital/nil/beans/Indicator.java

```diff
@@ -68,6 +69,10 @@ public Calendar getAssuranceDate() {
         return getProperty(PropertyKeys.ASSURANCE_DATE);
     }
 
+    public boolean hasAttachments() {
+        return getAttachments() != null || !StringUtils.isEmpty(getDetails().getTechnicalSpecificationUrl()) || !StringUtils.isEmpty(getDetails().getQualityStatementUrl());
```

Comment:
Think you want `isEmpty(getAttachments())` as the prototype document will give you an empty list by default, not a null one.